### DISCUSSION TITLE
REST API documentation not being generated?

### DISCUSSION THREAD

**Discussion by aaabdallah** [keycloak/keycloak#8913] [Q&A]
Dear all,
When I cloned the keycloak-documentation repository and checked out tag 4.8.3.Final, I successfully ran mvn clean install, but found no documentation generated under api_documentation except a small index.html with no details.

Other sections (e.g. server installation) are successfully generated in their target directories.

I subsequently tried other versions, and got the same result.

Is the REST API raw content found inside keycloak-documentation? When I grep for phrases that I find in the online documentation, I am not finding them...

Any ideas?

**Comment by djeikyb** [comment]
Same problem. Stuck on version 4.4 for probably one more year and it'd be great to have the rest api docs, even if I have to build them myself.

**Comment by abstractj** [comment]
@djeikyb we have a GH issue for it https://github.com/keycloak/keycloak/issues/12133, also there was PoC for that https://github.com/keycloak/keycloak/pull/12313
